src/memberdef.cpp

```diff
@@ -2584,7 +2586,9 @@ bool MemberDefImpl::isDetailedSectionLinkable() const
          _isReimplements() ||
          (_countReimplementedBy() > 0) ||
          // has user comments
-         Doxygen::userComments
+         Doxygen::userComments ||
+         // has source files and all entities are assumed to be documented
+         (sourceBrowser && extractAll)
```

Comment:
As indicated in the issue already:
- this will only be OK for HTML
- I don't think the `EXTRACT_ALL` is necessary
- I'm currently working on a correct solution